jfrog-tasks-utils/utils.d.ts

```diff
@@ -20,6 +20,9 @@ declare module '@jfrog/tasks-utils' {
     export function handleSpecFile(cliCommand: string, specPath: string): string;
     export function addBoolParam(cliCommand: string, inputParam: string, cliParam: string): string;
     export function addStringParam(cliCommand: string, inputParam: string, cliParam: string, require: boolean): string;
+
+    export function addIntParam(cliCommand: string, inputParam: string, cliParam: string): string;
+
```

Comment:
Let's be consistent with the no-space convention of this file
```suggestion
    export function addIntParam(cliCommand: string, inputParam: string, cliParam: string): string;
```